python/python/glide/async_commands/core.py

```diff
@@ -5961,3 +5961,60 @@ async def lcs_idx(
             Mapping[str, Union[list[list[Union[list[int], int]]], int]],
             await self._execute_command(RequestType.LCS, args),
         )
+
+    async def lpos(
+        self,
+        key: str,
+        element: str,
+        rank: Optional[int] = None,
+        count: Optional[int] = None,
+        max_len: Optional[int] = None,
+    ) -> Union[int, list[int], None]:
+        """
+        Returns the index or indexes of element(s) matching `element` in the `key` list.
+
+        See https://valkey.io/commands/lpos for more details.
+
+        Args:
+            key (str): The name of the list.
+            element (str): The value to search for within the list.
+            rank (Optional[int]): The rank of the match to return.
+            count (Optional[int]): The number of matches wanted. A `count` of 0 returns all the matches.
+            max_len (Optional[int]): The maximum number of comparisons to make between the element and the items
+                                     in the list. A `max_len` of 0 means unlimited amount of comparisons.
+
+        Returns:
+            If the element is found, its index (the zero-based position in the list) is returned.
```

Comment:
To follow general doc style
```suggestion
            Union[int, list[int], None]: If the element is found, its index (the zero-based position in the list) is returned.
```